R/ggplotMDS.R

```diff
@@ -70,22 +73,66 @@ ggplotMDS <- function(DGEdata,
                       sizeBy,
                       top = Inf,
                       labels,
-                      labelSize,
+                      labelSize = 3,
                       title,
                       hlineIntercept,
                       vlineIntercept,
-                      reflineColor = "darkgoldenrod1",
+                      reflineColor = "red",
                       reflineSize = 0.5,
-                      symShape = 16,
-                      symSize = 5,
-                      alpha = 0.7,
-                      shapes,
-                      colors,
+                      symShape = "circle",
+                      symSize = 10,
+                      transparency = 0.7,
                       dim.plot = c(1, 2)) {
 
-    # Default labels to colnames of DGEdata
+
+    assertthat::assert_that(!missing(DGEdata),
+                            !is.null(DGEdata),
+                            class(DGEdata) %in% c("DGEobj", "DGEList", "matrix", "Dataframe"),
```

Comment:
**Dataframe** should be **data.frame**

Reply:
I found another issue with matrix type, I suggest updating the assert to:
`                            all(class(DGEdata)) %in% c("DGEobj", "DGEList", "matrix", "array", "data.frame"),
`